Makd.mak

```diff
@@ -651,11 +651,10 @@ d2conv: $O/d2conv.stamp
 
 $O/d2conv.stamp: $C
 	$Vfind $C -type f -regex '^.+\.d$$' > $@
-ifeq "$(shell d1to2fix --help | grep -- --input)" ""
+	@if d1to2fix --help | grep -- --input; then \
+	$(call exec, d1to2fix $(DIMPORTPATHS) --fatal --input=$@) \
+	@else \
```

Comment:
Without `V=1` I get `/bin/sh: 4: Syntax error: end of file unexpected (expecting "fi")`. This is definitely broken, the statement before `else` also must end with `;` when its in one line. I don't know because of what black magic is working for you, but this is definitely broken.